web-ui/src/main/resources/catalog/components/elasticsearch/EsFacet.js

```diff
@@ -232,16 +232,16 @@
             },
             indexingErrorMsg: {
               terms: {
-                field: "indexingErrorMsg",
+                field: "indexingErrorMsg.type",
                 size: 10,
-                exclude: "Warning.*"
+                exclude: "warning"
```

Comment:
I spotted these too @fxprunayre , have to be modified too I reckon?

Reply:
I would replace `indexingErrorMsg` and `indexingWarningMsg` by 
```
indexingErrorMsg: {
              terms: {
                field: "indexingErrorMsg.string",
                size: 10
              }
            }
```
in order to display each types of error or warning to easily filter on it

![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/d58021d9-5aaa-432f-b350-d1504ff2914f)

What do you think?

Reply:
That makes sense to me. I have pushed your proposed change, but could not verify that these facets are actually shown. 

How to configure the search / editor board to show this facet? (I do not see "error types" and "indexing errors" as shown in your screenshot).

Reply:
This facet is displayed in admin > status - in the indexing error block

Reply:
Excellent, thanks!

Reply:
Verified this is working. The only drawback here is that the values are not replaced in the `indexingErrorMsg.string` template should they be present. But doing that would also create a lot of duplicate facet values in the list. Perhaps best to keep it to the template string then.